etc/dbus-serialbattery/config.default.ini

```diff
@@ -12,6 +12,10 @@ LOGGING = INFO
 MAX_BATTERY_CHARGE_CURRENT    = 50.0
 MAX_BATTERY_DISCHARGE_CURRENT = 60.0
 
+; ----------Cell count ------------
+; Predefine cell count for Jkbms_can
+; Should be autodetected if not set
+JKBMS_CAN_CELL_COUNT = 1
```

Comment:
Please move this under `; --------- BMS specific settings ---------`

Reply:
done